libs/portals/admin/ids-admin/src/shared/components/ContentCard.tsx

```diff
@@ -57,25 +124,108 @@ const ContentCard: FC<ContentCardProps> = ({
   useEffect(() => {
     originalFormData.current = new FormData(ref.current as HTMLFormElement)
   }, [ref])
+
   return (
-    <Box
-      borderRadius="large"
-      paddingY={2}
-      paddingX={4}
-      display="flex"
-      flexDirection="column"
-      justifyContent="spaceBetween"
-      height="full"
-      width="full"
-      border={'standard'}
-    >
-      <Box>
-        <Text marginTop={2} marginBottom={4} variant="h3">
-          {title}
-        </Text>
-        {description && <Text marginBottom={4}>{description}</Text>}
-      </Box>
-      <Form ref={ref} onChange={onChange} method="post">
+    <Form ref={ref} onChange={onChange} method="post">
+      <Box
+        borderRadius="large"
+        paddingY={2}
+        paddingX={4}
+        display="flex"
+        flexDirection="column"
+        justifyContent="spaceBetween"
+        height="full"
+        width="full"
+        border="standard"
+        position="relative"
+      >
+        <Box>
+          <Box
+            className={styles.title}
+            display="flex"
+            justifyContent="spaceBetween"
+            alignItems="baseline"
+          >
+            <Text ref={titleRef} marginTop={2} marginBottom={4} variant="h3">
+              {title}
+            </Text>
+            {intent !== 'none' && (
+              <Box>
+                <DropdownMenu
+                  title="Sync"
```

Comment:
This should be translated. Also, shouldn't this be
```
title={inSync ? 'Synced' : 'Sync'}
```
or something like that?

Reply:
you are right, it should be "Synced" or "Out of sync" according to Figma, I will make the changes